.github/workflows/update_notebook_links.py

```
@@ -82,7 +82,11 @@ def fix_links_in_notebook(notebook_path: str) -> int:
     relative_notebook_path = os.path.relpath(notebook_path, start=os.getcwd()).lower()
```

Ruff (0.12.2)

`os.getcwd()` should be replaced by `Path.cwd()`

(PTH109)